app/form_models/publishers/job_listing/about_the_role_form.rb

```diff
@@ -41,34 +41,48 @@ def organisation_type
   end
 
   def school_offer_presence
-    return if school_offer.present?
+    return if remove_html_tags(school_offer).present?
 
-    if organisation_type == "schools"
-      errors.add(:school_offer, I18n.t("about_the_role_errors.schools_offer.blank", organisation: organisation_type))
-    else
-      errors.add(:school_offer, I18n.t("about_the_role_errors.school_offer.blank", organisation: organisation_type))
-    end
+    errors.add(:school_offer, :blank, organisation: organisation_type)
```

Comment:
I'm wondering if we can get rid of `organisation_type` for the error message - the label originally changed depending on the organisation type but we changed it back to say `What does your school offer?` for all orgs - maybe we should do the same with the error message.

Reply:
Something for a content designer perhaps. Will drop a message to a content designer to see what they think.